packages/server/src/api/controllers/layout.ts

```diff
@@ -1,9 +1,9 @@
 import { EMPTY_LAYOUT } from "../../constants/layouts"
 import { generateLayoutID, getScreenParams } from "../../db/utils"
 import { events, context } from "@budibase/backend-core"
-import { BBContext, Layout } from "@budibase/types"
+import { BBContext, Layout, UserCtx } from "@budibase/types"
 
-export async function save(ctx: BBContext) {
+export async function save(ctx: UserCtx<Layout, Layout>) {
```

Comment:
Similar to above comment - just for clarity we would normally have a type alias for `LayoutRequest` and `LayoutResponse` - NAB at all - just the convention we've normally stuck with.